README.md

```diff
@@ -491,6 +495,12 @@ default version of the stub is Python 3.8.
 provide the path to the tar file in the `EXECUTION_ENV_PATH` in the
 `config.pbtxt` of all the models that want to use the execution environment.
 
+4. If you need to compile the Python backend stub, it is recommended that you compile 
+it in the official Triton NGC containers. Otherwise, your compiled stub may use
+dependencies that are not available in the Triton container that you are using for
+deployment. For example, compiling the Python backend stub in Ubuntu 16.04 while
```

Comment:
"For example, compiling the Python backend stub on an OS other than Ubuntu 20.04 can lead to unexpected errors."